sb.web.app/src/locales/en.json

```diff
@@ -64,7 +64,9 @@
         "no_users_found": "No members found for '{searchWord}'.",
         "send_token": "Send {token}",
         "commentLabel": "Comment",
-        "unknown_city": "Unknown city"
+        "unknown_city": "Unknown city",
+        "addMember": "Add Member",
```

Comment:
Inconsistency of these keys is a problem - can we agree on snake case 'add_member', 'add_category' etc?

No need to refactor all the existing ones at this time - but maybe we can refactor the ones we are touching